api/symfony.lock

```diff
@@ -6,17 +6,17 @@
         "version": "v1.8.1"
     },
     "api-platform/core": {
-        "version": "2.5",
+        "version": "3.2",
         "recipe": {
             "repo": "github.com/symfony/recipes",
-            "branch": "master",
-            "version": "2.5",
-            "ref": "a93061567140e386f107be75340ac2aee3f86cbf"
+            "branch": "main",
+            "version": "3.2",
+            "ref": "696d44adc3c0d4f5d25a2f1c4f3700dd8a5c6db9"
         },
         "files": [
             "config/packages/api_platform.yaml",
             "config/routes/api_platform.yaml",
-            "src/Entity/.gitignore"
+            "src/ApiResource/.gitignore"
```

Comment:
If you deleted this new directory, should we revert this change as well?

Reply:
I don't know if this section tracks the files that *may be* changed by the recipes or the files that *were* changed.
So i left it here, but decided against an additional empty directory for now.